### PR TITLE
CI improvement DLC [RDY]

### PR DESCRIPTION
**Pull request by JAremko**
Automatically publish docs to develop.spacemacs.org on push to master via PR from unprivileged account.

*Example: https://github.com/syl20bnr/develop.spacemacs.org/pull/15*

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,32 +1,32 @@
...
+  - TESTS=core EVM_EMACS=emacs-24.5-travis
+  - TESTS=core EVM_EMACS=emacs-25.1-travis
+  - TESTS=core EVM_EMACS=emacs-git-snapshot-travis
+  - TESTS=doc EVM_EMACS=emacs-24.4-travis
+  - TESTS=doc EVM_EMACS=emacs-24.5-travis
+  - TESTS=doc EVM_EMACS=emacs-25.1-travis
+  - TESTS=doc EVM_EMACS=emacs-git-snapshot-travis
+  - PUBLISH=html EVM_EMACS=emacs-25.1-travis
+  global:
+    - secure: OcM55oF/X9s4gutRVRohp6gNjsUUzZ9oad1O0Ve52sa23ii/cT6M//g0bSX+q/5DgzqUCySeat6QgPy1lHIQTOAe9TNTwqbt0CNN10k1vxiw+ufg0kzhWl4/AGzgqaxdtmkWJSdelnKOG2wgWGnHekiv8gtEw1PPAyktaDCK8/0CSf/XC3ZpWNbomv2HX6yRWYiVwUCftkT5BUJKssMcvlMFdMMoxGsb0FmzEnXba88lxBQ/bM8ltWWSDW9nRShzyR2Cow2RmEZ2fwgAggbQEjaQl+NOuN91ERLVpSwMFOI+pv+i/WHJve+Wr8QyrlWnvN7+bXt6rODw4s17tArXWJwjZdUwmRB7O0cUuPhSxfr2hLNzVfkHduQ1TgJ9ZJ8SXgGJ94tnlQeYVaKS+NgJWpr/Ihgvps3IXEsZ4QRY6xZPkkwKWbzgEV6gg1ogz0u7p7rLOwBadFpkraRPzb1bA6Okrg3VDbbd0+kOTowGfsijb1DTihFqSqbMWkq6nxlmJoAOjIuV/Y9MCCqux7C2SxutY5zXPqiVnee/L2IY/Sv/BJk5p3ZKAjjXKA5dbCZ5nENj5WQ01qX7Lkp6QUwsXMQrVZic8XwFnXbbBp0OumY6NBAcj3VBsjB2wshPpPMk+qtSNoxwDppkRGVJai/bTThstdSkoyEd+N+DaYl4XFk=
```
Comment (JAremko) [inline .travis.yml]: This need to be replaced by @syl20bnr  `cd spacemacs &&  travis encrypt THEBBOT_TK=<TOKEN>`

File: .travis-build.sh [hunk 2]
```diff
@@ -73,6 +74,81 @@ if [ ! -z "$FORMATTING" ]; then
 		;;
 	esac
 fi
+
+# If we are pushing changes to the master branch,
+# open PR to syl20bnr/develop.spacemacs.org with Spacemacs
+# documentation exported as HTML and formatted with spacefmt
```
Comment (syl20bnr) [inline .travis-build.sh]: Actually one of the goals for `develop.spacemacs.org` is to be up to date with the develop branch of 
Spacemacs (the other role is website related development waiting for a new major release).

Comment (JAremko) [inline .travis-build.sh]: @syl20bnr ok I'll make the bot open PRs to develop and master (based on which branch is updated)

File: .travis-build.sh [hunk 3]
```diff
@@ -73,6 +79,81 @@ if [ ! -z "$FORMATTING" ]; then
...
+		echo "beep beep boop... Beep?" >> msg
+		/tmp/hub pull-request -F msg
+		printf '=%.0s' {1..70}
+		printf "\n DONE!\n"
+		printf '=%.0s' {1..70}
+		echo
+		exit 0
+	else
+		printf '=%.0s' {1..70}
+		printf "\n NOTING TO COMMIT!\n"
```
Comment (zestyr) [inline .travis-build.sh]: typo

Comment (JAremko) [inline .travis-build.sh]: @zetok  thx

**Comment by JAremko** [comment]
Fixing links makes me want to commit sudoku :sob:

**Comment by syl20bnr** [comment]
Nice work as always 💜 
I setup a bot account named `emacspace`. I received some emails.
Will have to merge it in master.

Thank you !
Cherry-picked into develop branch, you can safely delete your branch.

**Pull request closed without merging**